src/audit/template_injection.rs

```diff
@@ -3,17 +3,19 @@
 //! This looks for job steps where the step contains indicators of template
 //! expansion, i.e. anything matching `${{ }}`.
 //!
-//! The following steps are currently supported:
-//! * `run:`, indicating template expansion into a shell script or similar
-//! * `actions/github-script`, indicating template expansion into a JavaScript function
+//! The list of injection sinks is based on [CodeQL's models](https://github.com/github/codeql/blob/main/actions/ql/lib/ext).
+//! The models are processed and filtered by a workflow on the zizmor repository.
```

Comment:
Note: we should also include some kind of attribution statement here (my read of the MIT License is that one *technically* isn't needed because this is a wholesale transformation of data rather than a "copy or substantial portion," but IANAL).

Something like this should be fine:

```
Sinks derived from the CodeQL models are licensed by GitHub, Inc.
under the terms of the MIT License.
```